USEPA/mywaterway
# Feature/update icons

## Related Issues:
* https://app.breeze.pm/projects/100762/cards/3165768

## Main Changes:
* Updates fishing icon (renamed "Eating Fish")
* Adds new "Aquatic Life" icon

## Steps To Test:
1. Visit homepage (notice four button/link icons)
2. From homepage, click each button/link icon and verify icons are correct
1. Visit /community and verify icons are correct on "Explore Your Water" community intro
2. Perform a search, and ensure "Eating Fish" and "Aquatic Life" icons are correct when tabs are selected
3. Visit /state page, select a state, and ensure "Eating Fish" icon is correct (on State Water Quality Overview" tab)
